internal/config/api/api.go

```diff
@@ -194,7 +194,7 @@ func LookupConfig(kvs config.KVS) (cfg Config, err error) {
 	}
 
 	corsAllowOrigin := strings.Split(env.Get(EnvAPICorsAllowOrigin, kvs.Get(apiCorsAllowOrigin)), ",")
-	if len(corsAllowOrigin) == 0 {
+	if len(corsAllowOrigin) == 1 && corsAllowOrigin[0] == "" {
```

Comment:
```suggestion
	if len(corsAllowOrigin) == 1 && corsAllowOrigin[0] == "" {
```

This is not needed we should do it differently here.

```
corsList := env.Get(EnvAPICorsAllowOrigin, kvs.Get(apiCorsAllowOrigin))
if corsList == "" {
   corsAllowOrigin = []string{"*"} // defaults to '*'
} else {
   corsAllowOrigin = strings.Split(corsList, ",")
   for _, cors := range corsAllowOrigin {
       if cors == "" {
           return cfg, errors.New("invalid cors value")
       }
   }
} 
```